bin/publish_builds

```diff
@@ -5,11 +5,8 @@ echo -e "PULL_REQUEST: ${TRAVIS_PULL_REQUEST}\n"
 
 if [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$PUBLISH" == "true" ]; then
 
-  if [ "$EMBER_ENV" != "production" ]; then
-    DISABLE_JSCS=true DISABLE_JSHINT=true ember build --environment=production
```

Comment:
`yarn build` seems to replace this, however `yarn build` does not `DISABLE_JSCS=true DISABLE_JSHINT=true`, if I read this correctly.

Is this something to be addressed?

Reply:
Yes, but the fix is to remove all references to `DISABLE_JSCS` and `DISABLE_JSHINT` in other files...

Basically, they were environment variables that emberjs-build was aware of and made special, but since we migrated away from emberjs-build (sometime before Ember 2.10?) they no longer have meaning.